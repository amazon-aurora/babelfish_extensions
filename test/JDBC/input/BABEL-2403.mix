--psql
SELECT * FROM sys.babelfish_inconsistent_metadata();
GO

-- psql
ALTER SYSTEM SET babelfishpg_tsql.migration_mode = 'multi-db';
SELECT pg_reload_conf();
GO

-- psql
SELECT * FROM sys.babelfish_inconsistent_metadata();
GO

-- psql
ALTER SYSTEM SET babelfishpg_tsql.migration_mode = 'single-db';
SELECT pg_reload_conf();
GO
