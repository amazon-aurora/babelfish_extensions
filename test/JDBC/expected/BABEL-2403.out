--psql
SELECT * FROM sys.babelfish_inconsistent_metadata();
GO
~~START~~
varchar#!#varchar#!#varchar#!#text
~~END~~


-- psql
ALTER SYSTEM SET babelfishpg_tsql.migration_mode = 'multi-db';
SELECT pg_reload_conf();
GO
~~START~~
bool
t
~~END~~


-- psql
SELECT * FROM sys.babelfish_inconsistent_metadata();
GO
~~START~~
varchar#!#varchar#!#varchar#!#text
~~END~~


-- psql
ALTER SYSTEM SET babelfishpg_tsql.migration_mode = 'single-db';
SELECT pg_reload_conf();
GO
~~START~~
bool
t
~~END~~

